source/geod24/LocalRest.d

```diff
@@ -393,6 +398,13 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
         }
     }
 
+    // Ask the node to shut down. Kept here as we need access to `API` symbol.
+    private static struct ShutdownCommand
+    {
+        /// Any callback to call before the Node's destructor is called
+        void function (API) callback;
```

Comment:
I deliberately made it not `nothrow` because exceptions are already caught and logged in LocalRest. So it's nicer UX.

Reply:
Good point

Reply:
Maybe document what happens if one `throw` then ?